examples/notebooks/beam-ml/tfma_beam.ipynb

```diff
@@ -0,0 +1,669 @@
+      "source": [
+        "# Define TFMA evaluation config, including two model specs for the two models we want to compare\n",
+        "eval_config_compare = text_format.Parse(\"\"\"\n",
+        "  ## Model information\n",
+        "  model_specs {\n",
+        "    name: \"model_v1\"\n",
+        "    # For keras (and serving models) we need to add a `label_key`.\n",
+        "    label_key: \"output\"\n",
+        "    is_baseline: true\n",
+        "  }\n",
+        "  model_specs {\n",
+        "    name: \"model_v2\"\n",
+        "    # For keras (and serving models) we need to add a `label_key`.\n",
+        "    label_key: \"output\"\n",
+        "  }\n",
+        "\n",
+        "  ## Post training metric information. These will be merged with any built-in\n",
+        "  ## metrics from training.\n",
+        "  metrics_specs {\n",
+        "    metrics { class_name: \"ExampleCount\" }\n",
+        "    metrics { class_name: \"MeanAbsoluteError\" }\n",
+        "    metrics { class_name: \"MeanSquaredError\" }\n",
+        "    metrics { class_name: \"MeanPrediction\" }\n",
+        "  }\n",
+        "\n",
+        "  slicing_specs {}\n",
+        "\"\"\", tfma.EvalConfig())"
+      ],
+      "metadata": {
+        "id": "pB4aXUo45RAg"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from tfx_bsl.public import tfxio\n",
+        "\n",
+        "output_path_compare = 'evaluation_results_compare'\n",
+        "\n",
+        "eval_shared_models = [\n",
+        "  tfma.default_eval_shared_model(\n",
+        "      model_name='model_v1',\n",
+        "      eval_saved_model_path=model_path_v1,\n",
+        "      eval_config=eval_config_compare),\n",
+        "  tfma.default_eval_shared_model(\n",
+        "      model_name='model_v2',\n",
+        "      eval_saved_model_path=model_path_v2,\n",
+        "      eval_config=eval_config_compare),\n",
+        "]\n",
+        "\n",
+        "tfx_io = tfxio.TFExampleRecord(\n",
+        "          file_pattern=tfrecord_file,\n",
+        "          raw_record_column_name=tfma.ARROW_INPUT_COLUMN)\n",
+        "\n",
+        "# Run Evaluation\n",
+        "with beam.Pipeline() as pipeline:\n",
+        "    _ = (\n",
+        "        pipeline\n",
+        "        | 'ReadData' >> tfx_io.BeamSource()\n",
+        "        | 'EvalModel' >> tfma.ExtractEvaluateAndWriteResults(\n",
+        "           eval_shared_model=eval_shared_models,\n",
+        "           eval_config=eval_config_compare,\n",
+        "           output_path=output_path_compare))"
+      ],
+      "metadata": {
+        "id": "4eVdpW0aWTah"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Visualize results\n",
+        "results = tfma.load_eval_results(output_paths=output_path_compare)\n",
+        "tfma.view.render_time_series(results)"
```

Comment:
Right now, this doesn't render in GitHub - https://github.com/apache/beam/blob/50e2a440729ea19ae939584933d96e11b156bf6d/examples/notebooks/beam-ml/tfma_beam.ipynb (as compared to the results in https://github.com/apache/beam/blob/50e2a440729ea19ae939584933d96e11b156bf6d/examples/notebooks/beam-ml/run_inference_multi_model.ipynb for example)

I think if you run the notebook and upload it with the run results it should render - could you do that?

Reply:
I first committed with the output of the two visualisations. But they did not render on Github so I removed them. I can try again if you want but I don't know if/ how can can solve this

Reply:
I added screenshots for now. Suggestions on how to do it otherwise are still welcome of course

Reply:
LGTM - it looks like an extra block got added to the bottom of the notebook though, could you please remove it? Unfortunately the diff no longer renders, so a screenshot is the best I can do:

![image](https://user-images.githubusercontent.com/42773683/218588534-5a2f4f45-c14c-482d-9500-e26ee5c0918f.png)

Reply:
Ok thanks, I removed it